src/main/java/teammates/common/datatransfer/FeedbackMcqQuestionDetails.java

```diff
@@ -20,10 +20,22 @@
 import teammates.logic.core.StudentsLogic;
 
 public class FeedbackMcqQuestionDetails extends FeedbackQuestionDetails {
-    public int numOfMcqChoices;
-    public List<String> mcqChoices;
-    public boolean otherEnabled;
-    FeedbackParticipantType generateOptionsFor;
+    private int numOfMcqChoices;
+    private List<String> mcqChoices;
+    private boolean otherEnabled;
```

Comment:
@damithc, we shouldn't rename this just like that because this will affect the existing questions in the database, where they stored this field using the name `otherEnabled`.